pkg/kyverno/common/common.go

```diff
@@ -0,0 +1,149 @@
+package common
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	v1 "github.com/nirmata/kyverno/pkg/api/kyverno/v1"
+	"github.com/nirmata/kyverno/pkg/kyverno/sanitizedError"
+	"github.com/nirmata/kyverno/pkg/openapi"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+//GetPolicies - Extracting the policies from multiple YAML
+func GetPolicies(paths []string) ([]*v1.ClusterPolicy, error) {
+	var policies []*v1.ClusterPolicy
+	for _, path := range paths {
+		path = filepath.Clean(path)
+
+		fileDesc, err := os.Stat(path)
+		if err != nil {
+			return nil, err
+		}
+
+		if fileDesc.IsDir() {
+			files, err := ioutil.ReadDir(path)
+			if err != nil {
+				return nil, err
+			}
+
+			listOfFiles := make([]string, 0)
+			for _, file := range files {
+				listOfFiles = append(listOfFiles, filepath.Join(path, file.Name()))
+			}
+
+			policiesFromDir, err := GetPolicies(listOfFiles)
+			if err != nil {
+				return nil, err
+			}
+
+			policies = append(policies, policiesFromDir...)
+		} else {
+			getPolicies, getErrors := GetPolicy(path)
+			var errString string
+			for _, err := range getErrors {
+				if err != nil {
+					errString = errString + err.Error() + "\n"
+				}
+			}
+
+			if errString != "" {
+				return nil, errors.New(errString)
+			}
+
+			for _, policy := range getPolicies {
+				policies = append(policies, policy)
+			}
+		}
+	}
+
+	for i := range policies {
+		setFalse := false
+		policies[i].Spec.Background = &setFalse
+	}
+
+	return policies, nil
+}
+
+// GetPolicy - Extracts policies from a YAML
+func GetPolicy(path string) ([]*v1.ClusterPolicy, []error) {
+	clusterPolicies := make([]*v1.ClusterPolicy, 0)
+	errors := make([]error, 0)
+
+	file, err := ioutil.ReadFile(path)
+	if err != nil {
+		errors = append(errors, fmt.Errorf("failed to load file: %v", err))
```

Comment:
Just a continuation on error handling for CLIs - If we return and show errors to the user, there is no need to log.

Reply:
Done. Updated code accordingly.